react/week2/src/App.js

```diff
@@ -0,0 +1,114 @@
+import "./App.css";
+import { useState, useEffect } from "react";
+
+function TodoItem({ id, description, onDelete }) {
+  const [isChecked, setChecked] = useState(false);
+
+  const check = () => {
+    setChecked(!isChecked);
+  };
+
+  return (
+    <li style={{ 'textDecoration': isChecked ? 'line-through' : 'none' }}>
+      {id}- {description}
+      <input type="checkbox" onClick={check}></input>
+      <button onClick={() => onDelete(id)}>Delete</button>
+    </li>
+  );
+}
+
+function TodoList({ todos }) {
+  const [Todos, setTodos] = useState(todos);
+
+  const addTodo = () => {
+      const newTodo = {
```

Comment:
This works, but I think its generally a good idea to stick with the functional version of the state changing function.
See here:
https://stackoverflow.com/questions/56404819/usestate-hook-setstate-function-accessing-previous-state-value

Reply:
Thank you for your time and feedback